profiles/common/modules/features/nexteuropa_newsroom/classes/NewsroomAgenda.inc

```diff
@@ -0,0 +1,679 @@
+<?php
+
+/**
+ * @file
+ * Code for the NewsroomAgenda class.
+ */
+
+/**
+ * NewsroomAgenda class base class.
+ */
+abstract class NewsroomAgenda {
+
+  const AGENDA_VIEW_SCHEDULE = 'schedule';
+  const AGENDA_VIEW_MONTH = 'month';
+  const AGENDA_VIEW_WEEK = 'week';
+  const AGENDA_VIEW_DAY = 'day';
+  const AGENDA_VIEW_SCHEDULE_BLOCK = 'schedule-block';
+
+  protected $view = NewsroomAgenda::AGENDA_VIEW_SCHEDULE;
+  protected $topicUrl = 'all';
+  protected $topics = array();
+  protected $typeUrl = 'all';
+  protected $typeId = NULL;
+  protected $currentDate = NULL;
+  protected $nextDate = NULL;
+  protected $previousDate = NULL;
+  protected $currentItems = array();
+  protected $nextItems = array();
+  protected $result = array();
+  protected $query = NULL;
+  protected $itemsCount = 0;
+  protected $urlParts = array();
+  protected $displayCalendar = TRUE;
+  protected $isBlock = FALSE;
+
+  /**
+   * Getter.
+   */
+  public function getDisplayCalendar() {
+    return $this->displayCalendar;
+  }
+
+  /**
+   * Get minimal items number per agenda page.
+   *
+   * @return int
+   *   Number of items.
+   */
+  public function getItemsNumber() {
+    return variable_get('newsroom_agenda_page_items_number', 10);
+  }
+
+  /**
+   * Setter.
+   */
+  public function setDisplayCalendar($value) {
+    $this->displayCalendar = $value;
+  }
+
+  /**
+   * Getter.
+   */
+  public function getIsBlock() {
+    return $this->isBlock;
+  }
+
+  /**
+   * Setter.
+   */
+  public function setIsBlock($value) {
+    $this->isBlock = $value;
+  }
+
+  /**
+   * Getter.
+   */
+  public function getView() {
+    return $this->view;
+  }
+
+  /**
+   * Setter.
+   */
+  public function setView($value) {
+    $this->view = $value;
+  }
+
+  /**
+   * Getter.
+   */
+  public function getItemsCount() {
+    return $this->itemsCount;
+  }
+
+  /**
+   * Getter.
+   */
+  public function getUrlParts() {
+    return $this->urlParts;
+  }
+
+  /**
+   * Getter.
+   */
+  public function getTopicUrl() {
+    return $this->topicUrl;
+  }
+
+  /**
+   * Setter.
+   */
+  public function setTopicUrl($value) {
+    $this->topicUrl = !empty($value) ? pathauto_cleanstring($value) : 'all';
+  }
+
+  /**
+   * Getter.
+   */
+  public function getTypeUrl() {
+    return $this->typeUrl;
+  }
+
+  /**
+   * Setter.
+   */
+  public function setTypeUrl($value) {
+    $this->typeUrl = !empty($value) ? pathauto_cleanstring($value) : 'all';
+  }
+
+  /**
+   * Getter.
+   */
+  public function getCurrentItems() {
+    return $this->currentItems;
+  }
+
+  /**
+   * Setter.
+   */
+  public function setCurrentItems($value) {
+    $this->currentItems = $this->sortItems($value);
+  }
+
+  /**
+   * Getter.
+   */
+  public function getNextItems() {
```

Comment:
@tatarbj I have renamed the variable to upcoming items, probably it will be more clear, but again  it is a getter of this property.
According to documentation https://www.drupal.org/node/608152
_Methods and class properties should use lowerCamel naming. In Drupal 8, properties of configuration entities are exempt of these conventions. Those properties are allowed to use underscores._
Can you exactly is wrong? I'm sorry I don't understand.